pkg/catalog/install.go

```diff
@@ -51,10 +51,10 @@ func Install(cfg InstallConfig) error {
 		},
 		Spec: profilesv1.ProfileSubscriptionSpec{
 			ProfileURL: profile.URL,
-			Version:    filepath.Join(cfg.ProfileName, cfg.Version),
+			Version:    filepath.Join(cfg.ProfileName, profile.Version),
 			ProfileCatalogDescription: &profilesv1.ProfileCatalogDescription{
 				Catalog: cfg.CatalogName,
-				Version: cfg.Version,
+				Version: profile.Version,
 				Profile: cfg.ProfileName,
```

Comment:
why not change all these to be `profile.X` i think they all exist there?

doesn't really matter ofc, just wondering why this change is there

Reply:
this is because `profile` contains the actual version, e.g. `v0.1.0`. `cfg.Version` is "latest" when not set. I think your right they could all change to `profile.`

Reply:
ah makes sense